examples/get-started/pure-js/maplibre/app.js

```diff
@@ -50,6 +60,22 @@ const deckOverlay = new DeckOverlay({
       getSourceColor: [0, 128, 200],
       getTargetColor: [200, 0, 80],
       getWidth: 1
+    }),
+    new GeoJsonLayer({
```

Comment:
Instead of changing the example could you update the render tests with the new images?